### PR TITLE
added initial apt-get update

### PR DESCRIPTION
**Pull request by 0atman**
On first install, the first step is to update your sources, or nothing will be installable!

### CODE REVIEW

File: setup-system.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 set -e
 
+# do an update to start things off
+function apt_update() {
+    sudo apt-get update
```
Comment (nottrobin) [inline setup-system.sh:5]: Could we standardise on `apt` rather than `apt-get` where possible?

Comment (0atman) [inline setup-system.sh:5]: I will if you will

Comment (nottrobin) [inline setup-system.sh:5]: That's unnecessarily passive aggressive when you're proposing merges into my project. How about something like "You also have an apt-get on line  17, just to let you know"?

Comment (nottrobin) [inline setup-system.sh:5]: https://github.com/nottrobin/system-setup/commit/0bfe2d8e0e4f98d3c8878287382ef916c81828e1

**Comment by nottrobin** [comment]
I'm closing this as I've implemented it myself: https://github.com/nottrobin/system-setup/commit/ed1427d69d15949f2e2043285efd00f84a8fa187

Thanks for pointing out the need to update. Very useful.

**Pull request closed without merging**